File: trading/js/OrderBookData.js
var request = require("request");
/*
set parameters

symbol
    string, trading pair

precision
    string, sets the precision of the calculated result
*/
const symbol = "tETHUSD";
const precision = "R0"; //P0-P4, R0
// set url
const url = "https://api.deversifi.com/bfx/v2/book/" + symbol + "/" + precision;
// execute request and print result
request(url, function(error, response, body) {
    console.log(JSON.parse(body));
});

File: trading/python/OrderBookData.py
import requests
import json

# set the url
url = "https://api.deversifi.com/bfx/v2/book/{symbol}/{precision}"

"""
set parameters

symbol
    string, trading pair

precision
    string, sets the precision of the calculated result
"""
url = url.format(
    symbol="tETHUSD",
    precision="P0")  # P0-P4, R0

# standard headers
headers = {
    "Accept": "application/json",
    "Content-Type": "application/json"
}

# execute the request and load into dictionary
response = json.loads(requests.request(
    "GET", url, headers=headers).text)

# print the response from dictionary
print(json.dumps(response, indent=2))
